src/librbd/image/RefreshParentRequest.h

```diff
@@ -4,7 +4,6 @@
 #ifndef CEPH_LIBRBD_IMAGE_REFRESH_PARENT_REQUEST_H
 #define CEPH_LIBRBD_IMAGE_REFRESH_PARENT_REQUEST_H
 
-#include "include/int_types.h"
```

Comment:
"include/types.h" pulls in lots of unneeded things ... and it appears the goal of this PR is to remove these unneeded includes.  However, I'd say removing includes because you want to rely on other header files (which aren't designed to be catch-all) makes it more difficult to understand how header dependencies are being satisfied. 

TL;DR: I love all the cleanup of unused header files but I really don't like implicitly relying on other random header files (a la parent_types.h) to satisfy dependencies.